src/praisonai/praisonai/cli/features/serve.py

```
@@ -268,17 +286,45 @@ async def invoke_agents(request: Request, query_data: AgentQuery = None):
+                            if hasattr(agent, 'astart'):
+                                result = await agent.astart(query)
+                            elif hasattr(agent, 'start'):
+                                import asyncio
+                                loop = asyncio.get_event_loop()
+                                result = await loop.run_in_executor(None, agent.start, query)
+                            else:
+                                raise AttributeError(f"Agent {agent_name} has no start/astart method")
+                            return {"response": str(result)}
+                    except Exception as e:
+                        # Fall back to crew-based approach if individual agent fails
+                        pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)